src/main/java/pl/numerki/frontend/Main.java

```diff
@@ -47,29 +50,45 @@ public static void main(String[] args) {
         double rightCompartment = s.nextDouble();
 
         if (!ZeroPosition.checkDifferentValuesSign(assembledFunction, leftCompartment, rightCompartment)) {
-            System.out.println("Wartości funckcji w predziałach końcowych muszą mieć różne znaki");
+            System.out.println("Wartości funckcji w predziałach końcowych muszą mieć różne znaki.");
             return;
         }
 
-        System.out.println("Wybierz warunek zakonczenia \n" + getEnding());
+        System.out.println("Wybierz warunek zakonczenia: \n" + getEnding());
         int choose = s.nextInt();
+        double bisectionResult, secantResult;
         switch (choose) {
             case 1 -> {
-                System.out.println("Podaj liczbe iteracji");
+                System.out.print("Podaj liczbe iteracji: ");
                 int iterations = s.nextInt();
-                System.out.println("Bisekcja: " + Bisection.getZeroPositionsWithIterationCondition(
-                        assembledFunction, leftCompartment, rightCompartment, iterations));
-                System.out.println("Metoda siecznych: " + SecantMethod.getZeroPositionsWithIterationCondition(
-                        assembledFunction, leftCompartment, rightCompartment, iterations));
+                bisectionResult = Bisection.getZeroPositionsWithIterationCondition(
+                        assembledFunction, leftCompartment, rightCompartment, iterations);
+                secantResult = SecantMethod.getZeroPositionsWithIterationCondition(
+                        assembledFunction, leftCompartment, rightCompartment, iterations);
+                System.out.println("Bisekcja: " + bisectionResult);
+                System.out.println("Metoda siecznych: " + secantResult);
             }
             case 2 -> {
-                System.out.println("Podaj dokladnosc");
+                System.out.print("Podaj dokladnosc: ");
                 double epsilon = s.nextDouble();
-                System.out.println("Bisekcja: " + Bisection.getZeroPositionsWithPrecisionCondition(
-                        assembledFunction, leftCompartment, rightCompartment, epsilon));
-                System.out.println("Metoda siecznych: " + SecantMethod.getZeroPositionsWithPrecisionCondition(
-                        assembledFunction, leftCompartment, rightCompartment, epsilon));
+                bisectionResult = Bisection.getZeroPositionsWithPrecisionCondition(
+                        assembledFunction, leftCompartment, rightCompartment, epsilon);
+                secantResult = SecantMethod.getZeroPositionsWithPrecisionCondition(
+                        assembledFunction, leftCompartment, rightCompartment, epsilon);
+                System.out.println("Bisekcja: " + bisectionResult);
+                System.out.println("Metoda siecznych: " + secantResult);
             }
+            default -> throw new Exception();
```

Comment:
Why here is exception and before (line 41) there was sout. Please standaralize in all switches